tests/unit/fromdate-step-form.spec.ts

```diff
@@ -1,13 +1,32 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import Vuex from 'vuex';
+
 import FromDateStepForm from '@/components/stepforms/FromDateStepForm.vue';
 
-import { BasicStepFormTestRunner } from './utils';
+import { BasicStepFormTestRunner, setupMockStore } from './utils';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
 
 describe('Convert Date to String Step Form', () => {
   const runner = new BasicStepFormTestRunner(FromDateStepForm, 'fromdate');
   runner.testInstantiate();
   runner.testExpectedComponents({
     'columnpicker-stub': 1,
-    'inputtextwidget-stub': 1,
+    'autocompletewidget-stub': 1,
```

Comment:
```suggestion
    'AutocompleteWidget-stub': 1,
```

and idem for `InputTextWidget ` 😬
Well... I am being picking :p

Reply:
I pass this one, as this convention is not used elsewhere...